parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -423,21 +423,37 @@ getPrimaryNicIP() {
     echo "$ip"
 }
 
+addKubeletNodeLabel() {
+    local LABEL_STRING=$1
+    if grep -i "$LABEL_STRING" <<< "$KUBELET_NODE_LABELS" > /dev/null 2>&1; then
+        echo "kubelet node label $LABEL_STRING is already present, nothing to add"
+        return 0
+    fi
+
+    echo "adding label $LABEL_STRING to kubelet node labels..."
+    if [ -n "$KUBELET_NODE_LABELS" ]; then
+        KUBELET_NODE_LABELS="${KUBELET_NODE_LABELS},${LABEL_STRING}"
+    else
+        # node labels shouldn't ever be empty, but we guard against it to be safe
+        KUBELET_NODE_LABELS=$LABEL_STRING
+    fi
+}
+
 # removes the specified LABEL_STRING (which should be in the form of 'label=value') from KUBELET_NODE_LABELS
-clearKubeletNodeLabel() {
+removeKubeletNodeLabel() {
```

Comment:
nit: move this and `addKubeletNodeLabel` to cse_helpers? Might be useful in future for other cases

Reply:
moved